TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/Utils.java

```diff
@@ -0,0 +1,19 @@
+package org.opensearch.migrations.replay;
+
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+
+public class Utils {
+    public static <A, B> Collector<A, ?, B>
+    foldLeft(final B seedValue, final BiFunction<? super B, ? super A, ? extends B> f) {
```

Comment:
I think this would benefit from comment

Reply:
Kartik had asked for this too.  Here's what I put on it...
    /**
     * See https://en.wikipedia.org/wiki/Fold_(higher-order_function)
     */
I'm not sure that I can do a better job describing it than the image does.